cli/tests/cfncluster-unittest.py

```diff
@@ -0,0 +1,294 @@
+# Copyright 2013-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Import source
+from cfncluster import cfncluster
+
+try:
+    from StringIO import StringIO
+except ImportError:
+    from io import StringIO
+
+from moto import mock_ec2, mock_cloudformation, mock_s3, mock_autoscaling
+
+import logging
+import re
+import boto3
+import unittest
+import configparser
+import os
+import stat
+import json
+
+test_log_stream = StringIO()
+config_file = 'cli/tests/config'
+with open('cloudformation/cfncluster.cfn.json') as f:
+    json_data = json.dumps(json.load(f))
+
+
+def config_logger_test():
+    logging.basicConfig(stream=test_log_stream, level=logging.INFO)
+
+
+def setup_configurations():
```

Comment:
Is this method just to get a link to the cfncluster cloudformation template? If so you can just grab `template_url` from `config` object after you call: [#L26](https://github.com/awslabs/cfncluster/blob/develop/cli/cfncluster/cfncluster.py#L26)